tophat/2.1.1/Dockerfile

```diff
@@ -0,0 +1,44 @@
+#################################################################
+# Dockerfile
+#
+# Version:          1
+# Software:         tophat
+# Software Version: 2.1.1
+# Description:      TopHat is a fast splice junction mapper for RNA-Seq reads
+# Website:          https://ccb.jhu.edu/software/tophat/index.shtml
+# Tags:             Genomics
+# Provides:         tophat 2.2.1
+# Base Image:       biodckr/biodocker
+# Build Cmd:        docker build --rm -t biodckrdev/tophat 2.1.1/.
+# Pull Cmd:         docker pull biodckrdev/tophat
+# Run Cmd:          docker run --rm -it biodckrdev/tophat <options> <files>
+#################################################################
+
+# Set the base image to Ubuntu
+FROM biodckr/biodocker
```

Comment:
same remarks, should use biocontainers base image and use LABEL for metadata (+ license info)